docs/params.md

```diff
@@ -0,0 +1,44 @@
+  For example, a `TrustingPeriodFraction` of `0.5` would entail that the `TrustingPeriod` of clients to the provider will be `ProviderUnbondingPeriod / 2`, while the `TrustingPeriod` of clients to every consumer will be `ConsumerUnbondingPeriod / 2`.
+  Note that a client MUST be update within the `TrustingPeriod` in order to remain active. 
+  For more details, see the [IBC specification of Tendermint clients](https://github.com/cosmos/ibc/blob/main/spec/client/ics-007-tendermint-client/README.md).
+- `CCVTimeoutPeriod` is the period used to compute the timeout timestamp when sending IBC packets. 
+  For more details, see the [IBC specification of Channel & Packet Semantics](https://github.com/cosmos/ibc/blob/main/spec/core/ics-004-channel-and-packet-semantics/README.md#sending-packets).
+  `CCVTimeoutPeriod` may have different values on the provider and consumer chains. If a sent packet is not relayed within this period, then the packet times out, the CCV channel is closed, and the corresponding consumer removed.
+  The `CCVTimeoutPeriod` on the provider MUST be larger than `ConsumerUnbondingPeriod`. 
+  The `CCVTimeoutPeriod` on the consumer is initial set via the `ConsumerAdditionProposal` gov proposal to add the consumer. 
+- `InitTimeoutPeriod` is the maximum time duration the Channel Initialization subprotocol may execute, 
+  i.e., for any consumer chain, if the CCV channel is not established within `InitTimeoutPeriod` since the `ConsumerAdditionProposal` was handled (the client to the consumer was created), then the consumer chain is removed.
+- `VscTimeoutPeriod` is the maximum time duration between sending any `VSCPacket` to any consumer chain and receiving the corresponding `VSCMaturedPacket`, without timing out the consumer chain and consequently removing it.
+  `VscTimeoutPeriod` MUST be larger than the `ConsumerUnbondingPeriod`.
+- `BlocksPerDistributionTransmission` is the number of blocks between rewards transfers from the consumer to the provider. 
+- `TransferPeriodTimeout` is the period used to compute the timeout timestamp when sending IBC transfer packets from a consumer to the provider. If this timeout expires, then the transfer is attempted again after `BlocksPerDistributionTransmission` blocks. 
+  The `TransferPeriodTimeout` on the consumer is initial set via the `ConsumerAdditionProposal` gov proposal to add the consumer. 
+  The `TransferPeriodTimeout` SHOULD be smaller than `BlocksPerDistributionTransmission x avg_block_time`, to make it easier to reason about the distribution subprotocol.   
+- `SlashMeterReplenishPeriod` exists on the provider such that once the slash meter becomes not-full, the slash meter is replenished after this period has elapsed. The meter is replenished to an amount equal to the slash meter allowance for that block, or `SlashMeterReplenishFraction * CurrentTotalVotingPower`.
+
+## Non-time-based parameters
```

Comment:
@mpoke lmk if you want non-time based parameters included in this doc as well, I've added the throttle related ones below. Feel free to remove if you don't want them included. 

There's a few others we could add as well:
- distribution_transmission_channel
- provider_fee_pool_addr_str
- consumer_redistribution_fraction
- historical_entries

Reply:
That will be nice to have. We can add them later.